### ISSUE TITLE
fontawesome is not shown in IE 11 on windows 10

### ISSUE THREAD

**Issue by iflyfei** [FortAwesome/Font-Awesome#8472]
Environment:
  IE 11 Version: 11.63.10586.0, Windows 10
Steps: 
1. go to http://fontawesome.io/examples/ 

- expected result:
   see the icons
- actual result:
 cannot see any icon and dev tool throw exception
  CSS3111: @font-face encountered unknown error.

**Comment by wolfy1339** [comment]
Cannot reproduce myself. Please give more details

**Comment by kirkov** [comment]
Also looks good in my IE11 on Windows 10.

**Comment by tagliala** [comment]
Cannot reproduce. Closing here but feel free to comment if you have news

**Comment by rstmm** [comment]
I can reproduce on a Windows 10 tablet with IE 11 in Edge mode.
Dev console fires
"CSS3111: @font-face encountered unknown error." and no icons are shown.

No idea why this could happen...

**Comment by tagliala** [comment]
@rstmm thanks!

@iflyfei could you please check?

**Comment by iflyfei** [comment]
maybe it causes by windows group policy not sure

**Comment by jwellsnh** [comment]
In my situation, a windows group policy was the cause.  See this TechNet article:

[Block untrusted fonts in an enterprise](: https://technet.microsoft.com/en-us/library/dn985836(v=vs.85).aspx#Turn_on_and_use_the_Blocking_untrusted_fonts_feature)

**Comment by tagliala** [comment]
@jwellsnh could you please update [Troubleshooting Guide](https://github.com/FortAwesome/Font-Awesome/wiki/Troubleshooting)?

**Comment by diegosanz** [comment]
With Windows Firewall stopped (from services.msc), fontawesome doesn't work. If I turn on the service it works. This happens only on IE.
(Sorry for my bad english).

**Comment by tagliala** [comment]
> With Windows Firewall stopped (from services.msc), fontawesome doesn't work. If I turn on the service it works. This happens only on IE.

Cannot confirm, it works for me (Win10x64 / IE11 / Windows firewall manually stopped under services.msc)

**Comment by diegosanz** [comment]
Sorry, I searched more and I finally find [this](https://github.com/FortAwesome/Font-Awesome/wiki/Troubleshooting#fonts-not-rendering-properly-running-windows-10), so it's already reported.
Thank you for your time.

**Comment by tagliala** [comment]
@diegosanz 

> Sorry, I searched more and I finally find this, so it's already reported.
Thank you for your time.

Is that a default setting under some versions of windows 10?

**Comment by diegosanz** [comment]
> Is that a default setting under some versions of windows 10?

In my computer, the "[block untrusted font feature](https://technet.microsoft.com/itpro/windows/keep-secure/block-untrusted-fonts-in-enterprise#Turn_on_and_use_the_Blocking_untrusted_fonts_feature)" it's not stablised because I don't have any key on my registry called `MitigationOptions ` in `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Kernel\`. Like this, with Windows Firewall running, Fontawesome works perfectly on IE11.

I cannot confirm because I only use Windows on my work, but I think that this is the default setting. In my case, the problem appears when I join into the domain of my company that, I don't know why, it stops the service of Windows Firewall and Fontawesome doesn't works.

**Comment by tagliala** [comment]
maybe this is a group policy option which does not apply outside domains

**Comment by tbinggeli** [comment]
I get this error too. Even when looking at fontawesome.io/icons. 

I have brought this up before, but I am still unable to resolve the issue.

**Comment by tbinggeli** [comment]
https://github.com/FortAwesome/Font-Awesome/issues/8825

**Comment by jhboyo** [comment]
In my case, i added the following tag and it's resolved
<meta charset="UTF-8" http-equiv="X-UA-Compatible" content="IE=edge">

**Comment by Rmittan** [comment]
I know this is an old issue, our organisation came across this so could possibly help some one else. This bug would only affect IE11 for us on any OS. It would work fine over http but over https it would play up. We resolved it by commenting out the Pragma from within the web config of the website. I believe this would then allow for caching.

**Comment by tagliala** [comment]
@Qore5 thanks

This case is also documented in our troubleshooting guide: https://github.com/FortAwesome/Font-Awesome/wiki/Troubleshooting#im-hosting-fonts-on-my-server-and-icons-dont-show-up

It does not strictly depend on IE but the way fonts are serverd is also involved

**Comment by autumnWei** [comment]
Environment:
IE 11 Version: 11.371.16299.0CO, Windows 10
in this enviroment.it shows normal.
Environment:
IE 11 Version: 11.2068.14393.0CO, Windows 10
in this enviroment. it cant't shows normal.and have two errors.
1)css3111:@font-face encountered unkown error.
2)css3114:@font-face failed OpenType embedding permission check.Permission must be Installable.
Please tell me  why this may have happened？？

**Comment by twaelti** [comment]
We had a customer with this problem. The solution was to make sure that fontdrvhost.exe isn't blocked by the antivirus software (in this case Trend Micro OfficeScan  was the culprit).

**Comment by tagliala** [comment]
@smaheedhara we have another issue with IE11 and FA 5.7.0 SVG

#14580
